src/modules/FileLocksmith/FileLocksmithUI/Views/MainPage.xaml

```diff
@@ -101,23 +92,22 @@
             <Grid Visibility="{x:Bind ViewModel.IsLoading, Converter={StaticResource boolToVisibilityConverter}, Mode=OneWay}">
                 <ListView
                     x:Name="ProcessesListView"
-                    Grid.Row="1"
                     IncrementalLoadingThreshold="10"
                     ItemsSource="{x:Bind ViewModel.Processes}"
-                    SelectionMode="None">
+                    SelectionMode="None"
+                    Padding="12,0,12,12">
```

Comment:
As per design spec, we'd want to align the outer edges of the cards with the path file textblock and icon.